src/fidgets/farcaster/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { AuthenticatorManager } from "@/authenticators/AuthenticatorManager";
+import { HubError, Signer } from "@farcaster/core";
+import { err, ok } from "neverthrow";
+
+export const createFarcasterSignerFromAuthenticatorManager = async (
```

Comment:
Doesn't have to be part of this PR, but I think it would be nice to have a `useFarcasterSigner` hook available to use throughout the app (and in this fidget) that includes a `publishCast` method that we can use throughout the app.

Reply:
I just made this, since the code that generates signers and stuff is done a bunch of times. I will add some publish features to it soon, currently it returns `signer,` `isSignerLoading`, and `fid` (of the current user)